src/MFRC522.cpp

```diff
@@ -167,7 +167,9 @@ MFRC522::StatusCode MFRC522::PCD_CalculateCRC(	byte *data,		///< In: Pointer to
 	// TODO check/modify for other architectures than Arduino Uno 16bit
```

Comment:
update doc please

Reply:
Would you prefer to have `PCD_CalculateCRC` receive a `timeout` parameter with a default of `89` for backwards compatibility?

Reply:
i think there is no real use case for a new parameter?
just remove the `todo` because you solved it 👍

Reply:
and remove the loop comment to a more suitable comment thanks